include/drivers/i2c/sun6i-a31-i2c.h

```diff
@@ -8,6 +8,9 @@
 
 #include <i2c.h>
 
+#define I2C_NUM_PINS 2
+#define I2C_PINS     (struct gpio_pin *)(struct gpio_pin[I2C_NUM_PINS])
```

Comment:
The cast is not necessary. An array of `struct gpio_pin` is already a pointer to `struct gpio_pin`

Reply:
Done.